src/controller/CHIPDeviceController.cpp

```diff
@@ -2299,6 +2302,12 @@ CHIP_ERROR DeviceCommissioner::ParseICDInfo(ReadCommissioningInfo & info)
         }
     }
 
+    // Intentionally ignore errors since they are not mandatory.
+    mAttributeCache->Get<IcdManagement::Attributes::UserActiveModeTriggerHint::TypeInfo>(kRootEndpointId,
+                                                                                         info.litUserActiveModeTriggerHint);
+    mAttributeCache->Get<IcdManagement::Attributes::UserActiveModeTriggerInstruction::TypeInfo>(
+        kRootEndpointId, info.litUserActiveModeTriggerInstruction);
```

Comment:
This isn't exact. If the device is a lit, `UserActiveModeTriggerHint` is mandatory and `UserActiveModeTriggerInstruction` might be mandatory depending on what are the set bits of `UserActiveModeTriggerHint`.

Reply:
agree. thanks

Reply:
Updated, will return error when the device is LIT.